docs/render.md

```diff
@@ -69,3 +69,6 @@ Another difference is that `m.render` method expects a [vnode](vnodes.md) (or a
 The `m.render` module is similar in scope to view libraries like Knockout, React and Vue. It is approximately 500 lines of code (3kb min+gzip) and implements a virtual DOM diffing engine with a modern search space reduction algorithm and DOM recycling, which translate to top-of-class performance, both in terms of initial page load and re-rendering. It has no dependencies on other parts of Mithril and can be used as a standalone library.
 
 Despite being incredibly small, the render module is fully functional and self-sufficient. It supports everything you might expect: SVG, custom elements, and all valid attributes and events - without any weird case-sensitive edge cases or exceptions. Of course, it also fully supports [components](components.md) and [lifecycle methods](lifecycle-methods.md).
+
+#### meta-description
+Explanation of the function, usage, and reasoning behind m.render().
```

Comment:
> Documentation for m.render(), the function Mithril uses to efficiently turn virtual DOM into real DOM nodes.

Reply:
`m.render()` is half redundant with the page title. Maybe we should update them to `m.*` for the titles, then have the description as just

> The function Mithril uses to efficiently turn virtual DOM into real DOM nodes.

Reply:
I'd keep them unprefixed in the TOC though.